infrastructure/main.tf

```diff
@@ -0,0 +1,30 @@
+terraform {
+  required_providers {
+    google = {
+      source = "hashicorp/google"
+      version = "3.86.0"
+    }
+  }
+  # Run locally
```

Comment:
Do we need to manage remote state, i.e in the event that multiple people want to interact and make changes to this project/infrastructure?

Reply:
If this is a one-time-command then you are absolutely right, no need to maintain state!